packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -119,34 +121,30 @@ export function mapTreeFromNodeData(
 		return undefined;
 	}
 
-	const mapTree = nodeDataToMapTree(
-		data,
-		normalizedFieldSchema.allowedTypeSet,
-		schemaValidationPolicy,
-	);
+	const mapTree = nodeDataToMapTree(data, normalizedFieldSchema.allowedTypeSet);
```

Comment:
This used to validate schema. It did so poorly, since it validated before any defaults were applied, and used a recursive validator on every node, so each node got checked a number of times equal to its depth.

The validation was moved out of that code, and moved to below, improving both issues, simplifying the code, and making the issue with defaults much easier to see and document.